CMakeLists.txt

```diff
@@ -181,21 +180,17 @@ if(BUILD_TESTING)
         ${CMAKE_BINARY_DIR}/lib/libgmock.a)
   elseif(WIN32)
     # Make sure we are always bootsrapped with vcpkg on Windows
-    find_package(GTest CONFIG)
-    if(NOT GTest_FOUND)
+    find_package(GTest)
+    if(NOT (GTEST_FOUND OR GTest_FOUND))
```

Comment:
@ThomsonTan - this is where I got into strange state, that after your `CONFIG` change done 3 days ago - Visual Studio 2019 IDE just completely refuse running Google Tests. It does compile into .exe with GTest in it. But it ran ZERO tests. I have to under your change, it works well after, and appears like we still got all tests passing. We can take it offline, or discuss in a separate issue.

Reply:
I'll keep my changes for now since these pass all the tests and work well with vs2019 IDE.